packages/esm-patient-search-app/src/patient-search-bar/patient-search-bar.scss

```diff
@@ -33,6 +33,15 @@
   justify-content: center;
   flex-direction: row;
   align-items: center;
+  border: 1px solid $ui-04;
+}
+
+.patientSearchInput {
+  border: none;
+}
+
+.orangeBorder {
```

Comment:
searchInputActive or highlighted

Reply:
No, it's not when it's active or highlighted, it's only for the global search.

Reply:
Also I think `orangeBorder` is a fine name for a class which actually just adds an orange border